main.go

```diff
@@ -81,10 +81,23 @@ var (
 		time.Hour * 24}
 )
 
+func getTerminalSizeFallback() (int, int) {
+	// Try to use the environment variables set by some shells.
+	rows, _ := strconv.Atoi(os.Getenv("LINES"))
+	columns, _ := strconv.Atoi(os.Getenv("COLUMNS"))
+
+	if columns*rows == 0 {
+		// Well. This is the default for many terminals.
+		return 80, 25
```

Comment:
```suggestion
		return 80, 24
```
I believe that line count of 24 is more common.